### DISCUSSION TITLE
Image versions

### DISCUSSION THREAD

**Discussion by stbeldarborge** [Azure/ipam#337] [Q&A] [answered]
Any way to find out which version are available for the images in azure container registry?

**Comment by DCMattyG** [comment, marked as answer]
Hi @stbeldarborge, I'm not aware of a way to list all available versions for an Azure Container Registry.

The current available versions align 1:1 with the available Releases from the Azure IPAM GitHub:

- v3.0.0
- v3.1.0
- v3.2.0
- v3.3.0
- v3.4.0
- v3.5.0

Hope that helps!

**Comment by stbeldarborge** [comment]
I know you have a tag for `latest`, but do you have `dev` or `developer` or something as well? So that we can use that in our test env and `latest` in prod?
